pnpm-workspace.yaml

```diff
@@ -26,6 +26,7 @@ catalogs:
     '@google-cloud/storage': 7.16.0
     '@headlessui/tailwindcss': ^0.2.2
     '@headlessui/vue': ^1.7.23
+    '@playwright/test': 1.54.0
```

Comment:
This updates playwright from `1.49` → `1.54` which hopefully won't break anything, cc @hanspagel

Reply:
Those updates used to break CI (for some super strange reasons that I can’t remember), but if it’s green here, I’ll just take it. 👍